main/webapp/modules/core/externals/suggest/css/suggest-4_3.css

```diff
@@ -67,55 +67,36 @@ below rules into your own stylesheet to customize.
 
 */
 
-.fbs-pane,
-.fbs-flyout-pane,
-li.fbs-nomatch,
-a.fbs-more-link,
-li.fbs-selected,
-.fbs-flyout-images,
-#fbs-topic-image { 
-    background: #fff; 
-}
```

Comment:
Here you are removing not just flyout styling, but also the styling on a lot of other elements: `.fbs-pane`, for instance, should likely stay. See the syntax for CSS selectors: there is a difference between `.fbs-pane .fbs-flyout-pane` (which could be removed entirely) and `.fbs-pane, .fbs-flyout-pane` (which should be simplified to just `.fbs-pane`).